### DISCUSSION TITLE
Need details about Nextjs routs deployment worldwide.

### DISCUSSION THREAD

**Discussion by VahanDrnoyan** [vercel/community#1637] [Help] [answered]
### Summary

Hello,

Could I ask one question?

Am I able to push my api routes to serverless functions of Aws lambdas for all locations and data centres over the world. Or I should only choose one location.

Thank you! Regards.

**Comment by VahanDrnoyan** [comment]
And the same for main application code. Will it be available from nearest CDN or datacenter of any location. Or it will be available only from one location.

**Comment by amyegan** [comment, marked as answer]
The right setup will depend on the specifics of your project. Serverless Functions are deployed to a single region by default. Best to set them close to your data sources (such as your [database](https://vercel.com/docs/solutions/databases)) for optimal performance.

Edge Functions, on the other hand, are deployed globally. They automatically execute in the region nearest to the user who triggers them, but come with their own [limitations](https://vercel.com/docs/concepts/functions/edge-functions#limitations) to consider.

You can learn more here:
- [Setting Serverless Function Regions](https://vercel.com/docs/concepts/functions/serverless-functions/regions#)
- [Edge Functions Overview](https://vercel.com/docs/concepts/functions/edge-functions#)

**Comment by VahanDrnoyan** [comment]
Thank you for answer.

However i see this in your provided docs, so multiple datacenters also are available, thank you!

> Multiple Serverless Function regions are only allowed for Enterprise accounts
